react/src/lib/List/examples/CheckBoxForKeyDownEvents.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Checkbox } from '@momentum-ui/react';
+
+export default class CheckBoxForKeyDownEvents extends React.PureComponent {
```

Comment:
This should contain both state and any necessary components to show the example (i.e. List)

Reply:
Moved List, state and checkbox to this component. 
Renamed it to `ListWithEventBubbling`